lib/util/template/index.ts

```diff
@@ -18,6 +18,10 @@ handlebars.registerHelper('encodeBase64', (str: string) =>
   Buffer.from(str ?? '').toString('base64'),
 );
 
+handlebars.registerHelper('decodeBase64', (str: string) =>
+  Buffer.from(str ?? '', 'base64').toString('ascii'),
```

Comment:
```suggestion
  Buffer.from(str ?? '', 'base64').toString(),
```
I think default `utf8` encoding is a better choice 

https://nodejs.org/api/buffer.html#buftostringencoding-start-end